templates/repo/editor/edit.tmpl

```diff
@@ -15,7 +15,7 @@
 					{{range $i, $v := .TreeNames}}
 						<div class="breadcrumb-divider">/</div>
 						{{if eq $i $l}}
-							<input id="file-name" value="{{$v}}" placeholder="{{ctx.Locale.Tr "repo.editor.name_your_file"}}" data-editorconfig="{{$.EditorconfigJson}}" required autofocus>
+							<input id="file-name" maxlength="500" value="{{$v}}" placeholder="{{ctx.Locale.Tr "repo.editor.name_your_file"}}" data-editorconfig="{{$.EditorconfigJson}}" required autofocus>
```

Comment:
Why it's 500?

Reply:
https://github.com/go-gitea/gitea/blob/2e33671f2c1e98759e4fd2a90944c534cfdf5776/services/forms/repo_form.go#L757

The 500 maxlength is from TreePath